src/components/sidebar/nodeView/StopForm.tsx

```diff
@@ -195,6 +209,14 @@ class StopForm extends ViewFormBase<IStopFormProps, IStopFormState> {
         navigator.goTo(url);
     };
 
+    private redirectToStopArea = (areaId: string | undefined) => {
+        if (!areaId) return;
```

Comment:
Remove this line. Instead, if no stopArea is selected, ( i ) button should be disbled (button takes disabled?: boolean as props)